plugins/modules/homebrew.py

```diff
@@ -415,7 +415,7 @@ def _current_package_is_installed(self):
         if self.force_formula:
             cmd.append("--formula")
         rc, out, err = self.module.run_command(cmd)
-        if err:
+        if rc != 0:
             self.failed = True
             self.message = err.strip()
```

Comment:
How about adding a fallback message in case `err` happens to be empty:
```suggestion
            self.message = err.strip() or ("Unknown failure with exit code %d" % rc)
```

Reply:
Cool suggestion! I'll add it now.